test/core/event_engine/test_suite/client_test.cc

```diff
@@ -135,7 +144,8 @@ TEST_F(EventEngineClientTest, ConnectExchangeBidiDataTransferTest) {
   EXPECT_TRUE(client_endpoint != nullptr);
```

Comment:
I'm not sure how this slipped in previously, but let's clean this up now. Comparing unique_ptr directly with a nullptr is wonky ... it works, but relies on some hidden casting to bool, and it appears to be deprecated in c++20. See https://en.cppreference.com/w/cpp/memory/unique_ptr/operator_cmp. Also, doing direct comparisons like this in googletest, you want to use the family of `EXPECT_NE(a, b)` tests rather than `EXPECT_TRUE(a != b)` because errors like "expected: nullptr, actual: 0x12345" can be more useful than "expected: true, actual: false".

Consider instead `EXPECT_NE(client_endpoint.get(), nullptr)`.

Reply:
Fixed.